gradle/verifier.gradle

```diff
@@ -85,6 +89,16 @@ task conjureJavaJersey(type: JavaExec) {
     doFirst { delete "src/generatedJersey/java/*" }
 }
 
+task conjureJavaDialogue(type: JavaExec) {
+    main = "com.palantir.conjure.java.cli.ConjureJavaCli"
+    classpath = project(':conjure-java').sourceSets.main.runtimeClasspath
+    args 'generate',  "${-> configurations.verificationApi.singleFile}", 'src/generatedDialogue/java', '--dialogue'
+
+    inputs.file "${-> configurations.verificationApi.singleFile}"
+    outputs.dir "src/generatedDialogue/java"
+    doFirst { delete "src/generatedDialogue/java/*" }
+}
```

Comment:
can we hook this up to the first-class local codegen now? configuration containing the tgz file with a nice `builtBy` should work here?